### PR TITLE
Make payload type class a bit more specific

### PR DESCRIPTION
**Pull request by elefeint**
For consistency with `PubSubMessageSource`.

### CODE REVIEW

**Comment by artembilan** [comment]
Travis failure:
```
[ERROR] /home/travis/build/spring-cloud/spring-cloud-gcp/spring-cloud-gcp-pubsub/src/test/java/org/springframework/cloud/gcp/pubsub/integration/inbound/PubSubInboundChannelAdapterTests.java:19:8: Unused import - java.io.UnsupportedEncodingException. [UnusedImports]
```

File: spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/integration/inbound/PubSubInboundChannelAdapter.java [hunk 1]
```diff
@@ -56,7 +56,7 @@
 
 	private HeaderMapper<Map<String, String>> headerMapper = new PubSubHeaderMapper();
 
-	private Class payloadType = byte[].class;
+	private Class<?> payloadType = byte[].class;
```
Comment (artembilan) [inline spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/integration/inbound/PubSubInboundChannelAdapter.java]: With this you have missed to update Copyright.
Or are you going to use your magic script? 😄

Comment (elefeint) [reply spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/integration/inbound/PubSubInboundChannelAdapter.java]: Apparently, having a magic script is insufficient if one forgets to run it :|

**Comment by artembilan** [comment]
The "agrh" is not an appropriate commit message. It doesn't say "what?" and "why?".

Just my opinion, but would be great to follow best practice when we develop Open Source: https://chris.beams.io/posts/git-commit/

**Pull request merged**